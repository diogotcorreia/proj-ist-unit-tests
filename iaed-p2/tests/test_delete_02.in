set /inY_lpALYtG]b\J/iDjGvsFQlUvlCKE///@apMK^adAPYHdUx///fD_dhdlLq\ViAS\/kKwjBkNQEvRi@cA//]YRWp\QK@b@rhPN///jAAILASkyiGWRaq///gMRLIAK@VJ]dHOA///HttiYQvMwszjLdH 5015
find inY_lpALYtG]b\J/iDjGvsFQlUvlCKE/@apMK^adAPYHdUx
list inY_lpALYtG]b\J/iDjGvsFQlUvlCKE/@apMK^adAPYHdUx/fD_dhdlLq\ViAS\
print
print
find inY_lpALYtG]b\J/iDjGvsFQlUvlCKE/@apMK^adAPYHdUx/fD_dhdlLq\ViAS\/kKwjBkNQEvRi@cA/]YRWp\QK@b@rhPN
set /ucUVgG[v_kmTwKL///p@xMmVgEra@hvmM/FVYzUDQTAPQkKQ\//@mNNXJBR\UnXxtN/pfEIIcW_ph@QEC[///tKy^]g^cU]W[SMV//xRlWwhpi`WhQEvN//W`K\eoNeb^jc^Ar/xRLnRs\osTd`\vE//TEKhMd@`yYsSymH 287
set //Tgzg_oUFEH_wqDu///UF@AB^]]zPTgJ\n///vWTSdoRofm`gzho/@GwjhiZFFcSOSQq//CDZokDviQ]cXJkT///]YUotsUv[LGht[i///PzcgNRCn_fuWs]U 9332
set //L`h@HYtbgeSohiw 5183
set //ki`v_s`\KpAQemA///VbjxkNn]xxWyPyM///qYzcepoOyPckyKO///plFYx\TLJHVTu_Y//`CeoDudOmZQ\Qlh///bFIrLpAI[FGfXxN/\fdUkwFLZl[w^mP///TpOiqSq]U`mAzXi 9802
list
set /X`TS\Yd@wKd`wrQ/dEwemdrbwePdhDJ///BuPQtRuQFpZ@cM_///R`p[uFjlfQMJAh[/FcX\tfCSRrOZWjX//UuugDpVOQdmjpNF/BDnMF@b@aY@DZT^/IFlpo^cCA`j`wvT 4213
set ///LM[RWIBiDHDXmqA///^m@nkjqcToax@y\///XTBGtt]`JOWgElF/DopXQCJZhU\zFfy//[HNjUzU_SuseZhM///VTVProJVQWAPKyb//rh\ySV[HQRzaZEe//@sWcCanSSgEyW@n 52
delete inY_lpALYtG]b\J/iDjGvsFQlUvlCKE
set /Gh_t^pnTKtXF]BG/ikCRDwNwTXA_zDT//sDHOIoPT`vp]DeD/M[XyzRwSKYjpZvR///\vOYNsuQjhsMMki//vYh^I^KN\mM_ZWA//Kc[`lGjAYkuKBl_/HK_t`lgGYRSG@EN///ym[btiUurT@Gnj] 4718
set /fQViu@ffC_RtYsB///WEXtnwkYi[_v@k\///JDNMIJqCAWTvKqz//AWcf`\yNQeGfsHw//vV\YrSvLSNyHyBl//GpfSOxtqGWTjIq\//MNmc]q`vRIyMvTE///@dy\xomZvodqFg]///k^YeeZzDrm`UNjH/lgoMkOysmI]j]T[ 5758
delete
set /DfVdbbNExktZQwS///wzycQScIUqmyjIA/fJxVtnFpDJrPvEf//cxj[QBiNAtFkWlI///ruZasOpOqXNR_Av/apzQiFGBe`GdCL_//]MymksaYI^\tJuR///dkBsyFwPT_BtXbc/\BishUOy`TqcvRZ///cVj]eNnA`ADldPY 4312
set //R]NlqsmH^UzcL@h//UHSEmRK\IqjbH^T//Rsp[dBelZsYRgOo//LGbqab^JPVO[Qon//BN^_AbrQCoaEUKt///zGlNMIZ@bQAKRPg 5352
set ///NAdiwvMTkapKg_d/GZynaDqIiu_`jKP/vapg^\b]zI]EpA^/UsDFGS^lijGnQ@^///KXzGeG]Go@]GCsT///I@VEE_gbbbUVyft 4201
set ///dnnoRhzKqruNbhN/KKY@ZrVILNtFWhF///TLf\OojqNZnHXTO/BLPrA[pkJ^G[V[f//MI[ks[_xcSkTeZM///^pXUhsHrQDVCNtB//AK[hj_wIJxSF_tL/yoEkbI@XyxtlCLc/HsmwYJkJAM[bX__//XBWFLNRFRY\unry 3604
set //seVkLNBDn^Q]YPF//c]oTk`bSbQVj`]y/wRTJonImdDv\bRq///thpRAYAtOXGlmEW//eEJpvccHqskuPFW//_wFdduydXOPhydi//JA\GGJ[Tpg`HYpU 4287
set /rZeoZzh\QXSNPhX//eOUJ[MHQYbF[tvR///CFMyRzqLHTLu`KX//ETfWveKwdAFAnzS///ivOnwUFq_VOPYrV/L[HrGT_VKQZoUe]//qksrHgKEPZrt\GZ 5016
list Gh_t^pnTKtXF]BG/ikCRDwNwTXA_zDT/sDHOIoPT`vp]DeD/M[XyzRwSKYjpZvR/\vOYNsuQjhsMMki/vYh^I^KN\mM_ZWA/Kc[`lGjAYkuKBl_/HK_t`lgGYRSG@EN
set ///bEMZTHTTYo_nw\T//@uKadbehZNEploP 518
set //Oioc@WrmjyXMtEq//ET_NpNdjDWLW]tZ/eFmvmVZPgGWPB_k///SUaDmUdflfIm@gs//Lv^d`r`QAewykym 9459
delete ki`v_s`\KpAQemA/VbjxkNn]xxWyPyM/qYzcepoOyPckyKO/plFYx\TLJHVTu_Y/`CeoDudOmZQ\Qlh/bFIrLpAI[FGfXxN/\fdUkwFLZl[w^mP
list Gh_t^pnTKtXF]BG/ikCRDwNwTXA_zDT/sDHOIoPT`vp]DeD/M[XyzRwSKYjpZvR/\vOYNsuQjhsMMki
delete fQViu@ffC_RtYsB/WEXtnwkYi[_v@k\/JDNMIJqCAWTvKqz/AWcf`\yNQeGfsHw/vV\YrSvLSNyHyBl/GpfSOxtqGWTjIq\
list ki`v_s`\KpAQemA/VbjxkNn]xxWyPyM/qYzcepoOyPckyKO/plFYx\TLJHVTu_Y/`CeoDudOmZQ\Qlh
set //hNlT_MJLr@lfQaI//DVDUo^OsV]\Ze`f///TZiwWACKayWFM[v///NzUAxqZvW`IIdEH///TPfW_`trIV]FvA\/BZhlp_McdGz^Qp@//QUztUJwZZujZkkP///ptNArymyrsSJcPo///_ssP]WlzmblVBcC 7291
delete Gh_t^pnTKtXF]BG/ikCRDwNwTXA_zDT
set //vjEsDy`DFaNQ\^g///`GGzhXAwdQTSRUn 1365
set ///SyD\pj`lwDxDWwZ/EEeKW``^cO^J[ae//FmCRScdjNyPaMNc///lKcowKzXCWtMF[T///`JzpqQ]AUhbq`@V//ISEM\sxGClxtSGK 3195
delete LM[RWIBiDHDXmqA/^m@nkjqcToax@y\/XTBGtt]`JOWgElF/DopXQCJZhU\zFfy/[HNjUzU_SuseZhM/VTVProJVQWAPKyb
delete ki`v_s`\KpAQemA/VbjxkNn]xxWyPyM/qYzcepoOyPckyKO/plFYx\TLJHVTu_Y/`CeoDudOmZQ\Qlh/bFIrLpAI[FGfXxN
delete Gh_t^pnTKtXF]BG/ikCRDwNwTXA_zDT/sDHOIoPT`vp]DeD/M[XyzRwSKYjpZvR/\vOYNsuQjhsMMki/vYh^I^KN\mM_ZWA/Kc[`lGjAYkuKBl_
delete ucUVgG[v_kmTwKL/p@xMmVgEra@hvmM/FVYzUDQTAPQkKQ\
delete
delete
delete Gh_t^pnTKtXF]BG
delete
delete hNlT_MJLr@lfQaI/DVDUo^OsV]\Ze`f/TZiwWACKayWFM[v/NzUAxqZvW`IIdEH/TPfW_`trIV]FvA\/BZhlp_McdGz^Qp@/QUztUJwZZujZkkP/ptNArymyrsSJcPo
delete DfVdbbNExktZQwS/wzycQScIUqmyjIA/fJxVtnFpDJrPvEf/cxj[QBiNAtFkWlI/ruZasOpOqXNR_Av/apzQiFGBe`GdCL_
quit