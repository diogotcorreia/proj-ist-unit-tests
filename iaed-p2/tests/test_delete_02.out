no data
kKwjBkNQEvRi@cA
/inY_lpALYtG]b\J/iDjGvsFQlUvlCKE/@apMK^adAPYHdUx/fD_dhdlLq\ViAS\/kKwjBkNQEvRi@cA/]YRWp\QK@b@rhPN/jAAILASkyiGWRaq/gMRLIAK@VJ]dHOA/HttiYQvMwszjLdH 5015
/inY_lpALYtG]b\J/iDjGvsFQlUvlCKE/@apMK^adAPYHdUx/fD_dhdlLq\ViAS\/kKwjBkNQEvRi@cA/]YRWp\QK@b@rhPN/jAAILASkyiGWRaq/gMRLIAK@VJ]dHOA/HttiYQvMwszjLdH 5015
no data
L`h@HYtbgeSohiw
Tgzg_oUFEH_wqDu
inY_lpALYtG]b\J
ki`v_s`\KpAQemA
ucUVgG[v_kmTwKL
not found
not found
not found
not found
not found
not found
not found
not found
not found
not found
not found
not found
not found
